### PR TITLE
Move `sources` and `launch` packages from `ddr` repository

### PR DESCRIPTION
**Pull request by c71n93**
Related to #29

I started fixing dependencies from resolving `sources` and `launch` packages.

---

## PR-Codex overview
This PR focuses on refactoring the GraphBuilder class and adding integration tests. 

### Detailed summary
- Refactored GraphBuilder class to match the version from the ddr repository
- Added integration tests for the entire program output format
- Added TempDirectoryTest class for testing the generation of temporary directories for modified xmir files

> The following files were skipped due to too many changes: `src/main/kotlin/org/objectionary/deog/launch/Workflow.kt`, `src/main/kotlin/org/objectionary/deog/sources/Sources.kt`, `src/main/kotlin/org/objectionary/deog/sources/XslTransformer.kt`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by c71n93** [comment]
@Graur could you check this, please?

**Comment by c71n93** [comment]
@Graur could you check this, please?

File: src/test/kotlin/org/objectionary/deog/unit/graph/TestBase.kt [hunk 1]
```diff
@@ -29,6 +29,8 @@ import kotlin.test.assertEquals
 
 /**
  * Common interface for all test classes
+ *
+ * @todo #29:90m/DEV Integration tests needs to be added
```
Comment (Graur) [inline src/test/kotlin/org/objectionary/deog/unit/graph/TestBase.kt:33]: @c71n93 Looks like this puzzle description too short.

**Review by Graur — CHANGES_REQUESTED** [review]

File: src/test/kotlin/org/objectionary/deog/unit/graph/TestBase.kt (hunk 1 above)
Comment (c71n93) [reply src/test/kotlin/org/objectionary/deog/unit/graph/TestBase.kt]: @Graur Done. Please check.

**Review by Graur — APPROVED** [review]

**Comment by c71n93** [comment]
@yegor256 could you take a look, please?

**Comment by yegor256** [comment]
@rultor merge

**Comment by c71n93** [comment]
@Graur Could you check again, please. I fixed problem because of which rultor failed.

**Comment by Graur** [comment]
@c71n93 Just ask @yegor256 to run `merge` command once again

**Comment by c71n93** [comment]
@yegor256 could you merge it again, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**